framework/src/modules/chain/chain.js

```diff
@@ -152,6 +155,19 @@ module.exports = class Chain {
 			this.scope.bus = await createBus();
 			this.scope.logic = await initLogicStructure(this.scope);
 			this.scope.modules = await initModules(this.scope);
+			// TODO: Global variable forbits to require on top
+			const Loader = require('./loader');
```

Comment:
- I think `requires` would be better at the top level of the file
- giving that this method gets too extensive maybe create a method `loadDependencies()` that does the same but then returns the value and here it's only assigned to this?

Reply:
Let's do this loadDependencies at the end of the technical debt because it will change a lot